OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/OptimoveConfig.java

```diff
@@ -68,28 +75,57 @@ public enum InAppDisplayMode {
         PAUSED
     }
 
-    // Private constructor to discourage not using the Builder.
-    private OptimoveConfig() {
-    }
+    public enum Region {
+        EU("eu-central-2"),
+        US("us-east-1"),
+        DEV("uk-1");
+        private final String region;
 
-    private void setRegion(@Nullable String region) {
-        this.region = region;
-    }
+        Region(String region) {
+            this.region = region;
+        }
 
-    private void setApiKey(@Nullable String apiKey) {
-        this.apiKey = apiKey;
+        @NonNull
+        public String toString() {
+            return region;
+        }
     }
 
-    private void setSecretKey(@Nullable String secretKey) {
-        this.secretKey = secretKey;
+    public static class FeatureSet {
+        private enum Feature {
+            OPTIMOVE,
+            OPTIMOBILE
+        }
+        List<Feature> features = new ArrayList<>();
+        public FeatureSet withOptimove(){
```

Comment:
Formatting not working?

Reply:
it is, but not autoformatting

aslo if i format everything it will be +300 lines of changes. i can format in the very end. But if you reviewers dont care --> can do now

Reply:
Sure, let's have a formatting PR separately then